test/exe/BUILD

```diff
@@ -81,3 +81,24 @@ envoy_cc_test(
         "//test/test_common:utility_lib",
     ],
 )
+
+# Due to the limitiations of how Windows signals work
+# this test cannot be executed through bazel and it must
+# be executed as a standalone executable.
+# e.g.: `./bazel-bin/test/exe/win32_outofproc_main_test.exe`
+envoy_cc_test(
```

Comment:
I assume there's no explicit way to generate this test binary target only under windows.

Reply:
Based on bazel documentation 

>The manual tag keyword is treated specially. It marks the test_suite target as "manual" so that it will be ignored by the wildcard expansion and automated testing facilities.

So it should not have any negative side effect on the POSIX compilations units and it should be ignored. Also given that this test has the same deps as the other units in the compilation unit if you tried to compile the entire //test/exe build file it should be minimal overhead.